modelservice/pom.xml

```diff
@@ -34,8 +34,13 @@
     <description>Adaptive Alerting Model Service.</description>
 
     <properties>
+        <codahale.metrics.version>3.0.2</codahale.metrics.version>
         <jacoco-percentage>0.17</jacoco-percentage>
         <jacoco-branch-percentage>0.25</jacoco-branch-percentage>
+        <percolator-client.version>6.4.1</percolator-client.version>
+        <aws.sdk.version>1.11.126</aws.sdk.version>
+        <aws-signing-request.version>0.0.22</aws-signing-request.version>
```

Comment:
AA shouldn't have any AWS dependencies. I assume we are doing this for authentication and access. Can you help me understand what this looks like without AWS dependencies?

Reply:
aws-iam-auth-required is set to false as default.
The dependencies are only applicable for elastic search that would like to add AWS IAM based auth.

Reply:
OK. I'm going to leave this for now, without necessarily committing to it for the future.